### ISSUE TITLE
Closing issues on the rfcs repo

### ISSUE THREAD

**Issue by Manishearth** [rust-lang/core-team#26]
The [RFCs repo has a lot of open issues](https://github.com/rust-lang/rfcs/issues), but these are mostly ignored by the teams.

We should perhaps turn off issue-filing on the repo (maybe archiving existing issues by mass migrating them if necessary) and update the readme to redirect people to Internals.

**Comment by steveklabnik** [comment]
As someone who participated in the first migration... you fundamentally need to give people some scratch space. That could be internals, that could be here, whatever. 

Honestly, I personally don't think internals is really used seriously by the teams; so I'd go the other way if I were running everything. 

I don't super care about the specifics, just that some place like this should exist in some form.

**Comment by Manishearth** [comment]
Yeah, I think internals does function as this kind of scratch space, which the teams can pay attention to (but essentially ignore threads that are going off into weird territory).
